examples/transfer-to-eth.js

```diff
@@ -0,0 +1,83 @@
+const hermez = require('../dist/node/index.js')
+const {
+  EXAMPLES_WEB3_URL,
+  EXAMPLES_PRIVATE_KEY1,
+  configureEnvironment
+} = require('./constants.js')
+
+function getRandomPrivKey (length) {
+  var randomChars = 'ABCDEF0123456789'
```

Comment:
Please use `let` instead of `var`, just to follow same language style across the whole repository